odtp/dashboard/utils/storage.py

```diff
@@ -226,3 +229,35 @@ def storage_run_selection(execution_id, repo_url, commit_hash):
         app.storage.user["run_selection"] = json.dumps(run_selection)
     except Exception as e:
         ui.notify(f"storage update for run selection failed: {e}", type="negative")
+    
+def app_storage_is_set(value):
```

Comment:
todo: this function is only used by `get_from_storage` which isn't used any more. So also this function should be removed.